instrument/instrument.go

```diff
@@ -0,0 +1,103 @@
+package instrument
+
+import (
+	"io/ioutil"
+
+	"github.com/sirupsen/logrus"
+	"gopkg.in/segmentio/analytics-go.v3"
+)
+
+type Client interface {
+	identify(userID string, traits analytics.Traits) error
+	track(userID string, event string, properties analytics.Properties) error
+	page(userID string, name string, properties analytics.Properties) error
+	group(userID string, groupID string, traits analytics.Traits) error
+	alias(previousID string, userID string) error
+}
```

Comment:
if you don't export these methods then you effectively have an interface you can't implement in a different client.

Reply:
🤦 yes